aws-lc-rs/src/cipher/chacha.rs

```diff
@@ -52,6 +52,8 @@ pub(crate) fn encrypt_block_chacha20(
         counter,
     )?;
 
+    crate::fips::set_fips_service_status_unapproved();
```

Comment:
As I mentioned elsewhere, should we set "unapproved" even when the operation fails?

Reply:
See earlier response.